Index: src/it/ristoranteGruppo3/entities/portate/Portata.java
===================================================================
diff --git a/src/it/ristoranteGruppo3/entities/portate/Portata.java b/src/it/ristoranteGruppo3/entities/portate/Portata.java
--- a/src/it/ristoranteGruppo3/entities/portate/Portata.java	
+++ b/src/it/ristoranteGruppo3/entities/portate/Portata.java	
@@ -24,6 +24,7 @@
         this.descrizione = descrizione;
     }
 
+    //todo eliminare il secondo costruttore ed anche nei figli
     /**
      * Costruttore per tutte le portate che non necessitano di una descrizione
      * @param namePortata Nome della portata
Index: src/it/ristoranteGruppo3/entities/portate/PrimoPiatto.java
===================================================================
diff --git a/src/it/ristoranteGruppo3/entities/portate/PrimoPiatto.java b/src/it/ristoranteGruppo3/entities/portate/PrimoPiatto.java
--- a/src/it/ristoranteGruppo3/entities/portate/PrimoPiatto.java	
+++ b/src/it/ristoranteGruppo3/entities/portate/PrimoPiatto.java	
@@ -24,6 +24,7 @@
         super(namePortata, pricePortata, typeEnum,descrizione);
         this.tipologiaPrimo = tipologiaPrimo;
     }
+
     /**
      * Costruttore per tutte le portate che non necessitano di una descrizione
      * @param namePortata Nome della portata
@@ -59,10 +60,8 @@
      */
     public void printPrimoPiatto(){
         this.printPortataGenerica();
-        if (getTipologiaPrimo()!=null){
             System.out.print(" e' un primo piatto a base di"+" "+ tipologiaPrimo.name().toLowerCase());
             System.out.println("\n");
-        }
     }
 }
 
