Index: src/Portata.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package src;\r\n\r\nimport java.util.ArrayList;\r\nimport java.util.List;\r\n\r\npublic abstract class Portata implements IPortata {\r\n\r\n   private String namePortata;\r\n   private double pricePortata;\r\n\r\n   private String typePortata;\r\n\r\n    public Portata(String namePortata, double pricePortata,String typePortata) {\r\n        this.namePortata = namePortata;\r\n        this.pricePortata = pricePortata;\r\n        this.typePortata = typePortata;\r\n    }\r\n\r\n    public String getNamePortata() {\r\n        return namePortata;\r\n    }\r\n\r\n    public double getPricePortata() {\r\n        return pricePortata;\r\n    }\r\n\r\n    @Override\r\n    public String getTypePortata() {\r\n        return typePortata;\r\n    }\r\n\r\n    void printPortata(){\r\n\r\n        System.out.println(namePortata + \" \" + pricePortata);\r\n    }\r\n\r\n\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Portata.java b/src/Portata.java
--- a/src/Portata.java	(revision 09925bdee6f37e7b71187704f4e1445b0391744a)
+++ b/src/Portata.java	(date 1669226221882)
@@ -30,7 +30,6 @@
     }
 
     void printPortata(){
-
         System.out.println(namePortata + " " + pricePortata);
     }
 
